test/controllers/upsert/config-vars.test.js

```diff
@@ -0,0 +1,49 @@
+const ConfigVarsStep = require('../../../src/controllers/upsert/config-vars');
+const heroku = require('../../../src/heroku');
+
+const setupBeforeAndAfter = require('./setup');
+
+const SAMPLE_APP_ID = '11111111-2222-3333-4444-555555555555';
+const SAMPLE_PIPELINE_ID = '22222222-3333-4444-5555-666666666666';
+const SAMPLE_APP_NAME = 'owlzilla';
+const SAMPLE_PIPELINE_NAME = 'owlline';
+const SAMPLE_CONFIG_VARS = { VAR1: 'value1', VAR2: 'value2' };
+
+describe('#src/controllers/upsert/config-vars', () => {
+  setupBeforeAndAfter();
+
+  describe('checkPrereqs()', () => {
+    it('should run when the pipeline has 1 or more config vars', async () => {
+      heroku.getPipelineId = jest.fn(() => ({ id: SAMPLE_PIPELINE_ID }));
```

Comment:
I'm surprised you don't need to mock out `src/heroku` for these overrides to work inside the `config-vars` file.